docs/rules/quote-props.md

```diff
@@ -216,9 +216,35 @@ and the following patterns _stop_ being problems:
 ```js
 var x = {
     "100": 1
+};
+```
+
+A `computed` flag, with default value `false`, can also be used as a modifier for the `"as-needed"` mode. When it is set to `true`, unnecessary property name computations will be reported.
```

Comment:
>Eh, you know, I'm starting to think I was wrong about bundling this functionality with quote-props. :(

In all honesty, I felt the same when I was implementing this rule and writing the docs. I still feel like these should probably be in close proximity but may be a name change for the rule might be better?

As a stop-gap, I think `allowComputed` defaulting to true is also good. That said I think that's objectively clearer than `computed`.

Thinking again, `"as-needed", { computed: true }` actually makes sense. `computed` is confusing without that `as-needed` context.